core/hw/gdrom/gdromv3.cpp

```diff
@@ -322,6 +316,13 @@ u32 GetFAD(u8* data, bool msf)
 void libCore_gdrom_disc_change()
 {
 	gd_setdisc();
+	read_params = { 0 };
+	set_mode_offset = 0;
+	packet_cmd = { 0 };
+	read_buff = { 0 };
```

Comment:
```suggestion
	read_buff.cache_index = 0;
	read_buff.cache_size = 0;
	memset(read_buff.cache, 0, 2352 * 8192);
```
Any issue with resetting the struct variables explicitly like this? It fixes the msvc heap size issue for me. Not sure I fully understand why.